tailwind.config.js

```diff
@@ -255,6 +263,8 @@ module.exports = {
         '15px': '0.938rem',
         '26px': '1.625rem',
         '32px': '2rem',
+        12: '0.75rem',
+        32: '2rem',
```

Comment:
Please update code to use existing sizes, so as '32px' already exists, no need for the same without "px".
As other sizes have "px", rename 12 to 12px as well